js/client-chat-tournament.js

```diff
@@ -332,6 +332,15 @@
 					}
 					break;
 
+				case 'scouting':
+					if (data[0] === 'allow') {
+						this.room.$chat.append("<div class=\"notice tournament-message-scouting\">Scouting is now allowed (Tournament players can watch other tournament battles)</div>");
+					}
+					else if (data[0] === 'disallow') {
```

Comment:
@Zarel I keep forgetting about it, considering it's one of three (or so) differences between my own style and Showdown style.